src/components/dragelement/unhover.js

```diff
@@ -14,22 +14,25 @@ var getGraphDiv = require('../../lib/dom').getGraphDiv;
 
 var hoverConstants = require('../fx/constants');
 
-var unhover = module.exports = {};
+module.exports = {
```

Comment:
Wanna go one step further and always put the `module.exports` at the end of the file?

Reply:
I think that's OK to have that on top where one could easily view inputs and outputs of the module.

Reply:
Well, that's why exactly people don't like wasting time on syntax PRs like this one. If we can't add a test (e.g. an eslint rule), then how can we tell whether the "before" or the "after" is better?

Reply:
I don't think we should consider this as a syntax PR. I would say it is more related to the difference between using function declarations and function expressions at the run time.
Maybe @antoinerg could help investigate the benchmark diffs?
Plus in large files having a clear interface can help the developers have a quick view of the methods available.

Reply:
> I would say it is more related to the difference between using function declarations and function expressions at the run time.

> Maybe @antoinerg could help investigate the benchmark diffs?

Ok, well if you can detect a performance improvement, then YES I'm all for this PR. Before that happens, I have no interest in this PR. 

> Plus in large files having a clear interface can help the developers have a quick view of the methods available.

One could also argue: inputs (at the top) / new things (in the middle) / output (at the bottom) is easier to read. In addition, most files that have a `module.exports = { /* */ }` block currently, have it at the bottom of the file. But like I said before, these are simply opinion, so please let's not waste time arguing over this.